Sources/TuistLoader/SwiftPackageManager/SettingsMapper.swift

```diff
@@ -86,6 +86,8 @@ struct SettingsMapper {
                 swiftFlags.append("-enable-upcoming-feature \"\(setting.value[0])\"")
             case (.swift, .enableExperimentalFeature):
                 swiftFlags.append("-enable-experimental-feature \"\(setting.value[0])\"")
+            case (.swift, .swiftVersion):
+                swiftFlags.append("-swift-version \(setting.value[0])")
```

Comment:
I'm not sure, if the swift flag or a `SWIFT_Version` build setting here is better. Maybe the dedicated setting is easier to read in the Xcode project.

Reply:
The same evolution proposal I linked [above](https://github.com/tuist/tuist/pull/6730/files#r1759724763) mentions adding a `-language-mode` compiler flag that works the same as `-swift-version`, but I'm not sure if that piece has landed/is available for use yet.

Reply:
If we want to use the flag and not the build setting and this has landed we should use that yes. But maybe this then needs some more logic to check what Xcode is used. Because probably this breaks Xcode < 16 then, where `-language-mode` presumably isn't working.

Reply:
We can keep using `-swift-version` but I would add a comment here linking to the [proposal](https://github.com/swiftlang/swift-evolution/blob/main/proposals/0441-formalize-language-mode-terminology.md#swift-compiler-option) that this should change to `-language-mode` once we drop support for Xcode 15.